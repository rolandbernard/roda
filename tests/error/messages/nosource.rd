// test: Test style of error message
// build: $BINARY $(realpath --relative-to=. %) --messages=nosource 2> %.out || true

extern fn test(a: f64): int;

pub fn foo(a: int): int {
    return test(a);
}
