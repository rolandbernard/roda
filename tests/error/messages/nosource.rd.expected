error[E0008]: type error, conflicting types `int` and `f64`
     = tests/error/messages/nosource.rd:4:19-4:21: note: expecting `f64` because of this
     = tests/error/messages/nosource.rd:6:15-6:17: note: expecting `int` because of this
     = tests/error/messages/nosource.rd:7:17: conflicting types `int` and `f64` for this expression
